creator-node/src/logging.js

```diff
@@ -1,5 +1,8 @@
 const bunyan = require('bunyan')
 const shortid = require('shortid')
+const { createNamespace } = require('cls-hooked')
```

Comment:
i'm still not convinced we should use this, especially at node 10 given the performance considerations https://github.com/bmeurer/async-hooks-performance-impact/blob/master/README.md https://itnext.io/one-node-js-cls-api-to-rule-them-all-1670ac66a9e8. this is probably better in node 14 but a 1.5x penalty for this in node 10 seems questionable

Reply:
Ripping it out for now